bin/du

```diff
@@ -36,6 +36,9 @@ $blocksize = $ENV{BLOCKSIZE} if $ENV{BLOCKSIZE}; # use environment if present
 getopts('HLPacklrsx') or usage();
 
 $blocksize = 1024 if $opt_k;
+if ($blocksize =~ m/[^0-9]/ || $blocksize == 0) {
+  die "$0: unexpected block size: $blocksize\n";
```

Comment:
du on archlinux returns `1`, FreeBSD returns 64 (for `-B`), and so on. I'd prefer some value that isn't `die`'s 255.

Reply:
Ok, I've added a new commit and there are no longer any calls to die() in the du command.